src/Model/ReferencesTrait.php

```diff
@@ -185,6 +185,15 @@ public function refLink($link, $defaults = []): self
      */
     public function getRef($link): Reference
     {
+        // support for relationship traversing link, e.g contact/company/customers
```

Comment:
using string is quite antipattern - what is the typical usecase?

In the future, we should drop static strings in advance of refactorable concept - this is the same as declaring classes with string directly.

Reply:
It is a shorthand for
`$user->ref('contact')->ref('company')->ref('customers')`
You can still use it

Reply:
This is much better and easier to understand (you learn 10 elementary methods and you understand them, instead of shorthand, shorthand, ... in Scopes, it is a little bit different, as the alternative is much longer).

Reply:
Yes, nobody is preventing you from doing so